ludwig/utils/metric_utils.py

```diff
@@ -33,73 +30,17 @@ def dynamic_partition(data: Tensor, partitions: Tensor, num_partitions: int):
     return res
 
 
-def masked_correct_predictions(
-        targets: Tensor,
-        preds: Tensor,
-        targets_sequence_lengths: Tensor
-) -> Tensor:
+def masked_correct_predictions(targets: Tensor, preds: Tensor, targets_sequence_lengths: Tensor) -> Tensor:
     """
     Params:
         targets: 2D tensor
         preds: 2D tensor
     """
-    truncated_preds = preds[:, :targets.shape[1]]
-    padded_truncated_preds = F.pad(
-        truncated_preds, pad=[0, targets.shape[1] - truncated_preds.shape[1]])
+    truncated_preds = preds[:, : targets.shape[1]]
+    padded_truncated_preds = F.pad(truncated_preds, pad=[0, targets.shape[1] - truncated_preds.shape[1]])
     correct_preds = padded_truncated_preds == targets
 
-    mask = sequence_mask(
-        lengths=targets_sequence_lengths,
-        maxlen=correct_preds.shape[1],
-        dtype=torch.int32
-    )
-    _, masked_correct_preds = dynamic_partition(
-        data=correct_preds,
-        partitions=mask,
-        num_partitions=2)
+    mask = sequence_mask(lengths=targets_sequence_lengths, maxlen=correct_preds.shape[1], dtype=torch.int32)
+    _, masked_correct_preds = dynamic_partition(data=correct_preds, partitions=mask, num_partitions=2)
 
     return masked_correct_preds.type(torch.float32)
-
-
-# TODO(shreya): After sequence loss
```

Comment:
@justinxzhao would this commented out code be helpful for when you're migrating the sequence losses? Feel free to remove if not.

Reply:
Now that we have the tf-legacy branch, my plan was to reference that instead.